src/Exscript/protocols/Telnet.py

```diff
@@ -36,11 +36,12 @@ def _telnetlib_received(self, data):
         self._receive_cb(data)
         self.buffer.append(data)
 
-    def _connect_hook(self, hostname, port):
+    def _connect_hook(self, hostname, port, init_timeout):
         assert self.tn is None
         rows, cols = get_terminal_size()
         self.tn = telnetlib.Telnet(hostname,
                                    port or 23,
+                                   init_timeout     = init_timeout,
```

Comment:
Please just use self.init_timeout and don't add init_timeout as parameter to  _connect_hook.